tests/tests-core/src/main/java/com/gentics/mesh/search/ElasticSearchProviderTimeoutTest.java

```diff
@@ -59,7 +59,14 @@ public static void setupTimeoutServer() throws IOException {
 			}
 		});
 		server.rxListen().blockingGet();
+	}
 
+	@Test
+	public void testHugeBatch() throws IOException {
```

Comment:
What does this test actually prove? There is no assertion in here - apart from that creating 1200 users succeeds, but that is completely unrelated to ES indexing.

Reply:
It is created during a timeout, so the indexing process should remain stable. Again, the direct use case is not reproducible, due to its spontaneous nature.

Reply:
I'll check if I'm able to extend BulkOperatorTest. Which will still be kinda artificial...

Reply:
Have made it eventually, with an extension that I don't really like.